sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/Models/GetDigitalTwinsEventRouteOptions.cs

```diff
@@ -18,5 +18,8 @@ public partial class GetDigitalTwinsEventRouteOptions
         /// <summary> Provides vendor-specific trace identification information and is a companion to TraceParent. </summary>
         [CodeGenMember("Tracestate")]
         public string TraceState { get; set; }
+
+        // This reduces the scope of this property to private since users shouldn't set page size here, but on the pageable instance's .AsPages() method instead.
```

Comment:
This could be worded - "This is internal because users should not set page size here. It should be set on the pageable instances's .AsPages() method."